cmd/podman/spec.go

```diff
@@ -319,11 +334,6 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		}
 	}
 
-	// HANDLE CAPABILITIES
-	if err := setupCapabilities(config, configSpec); err != nil {
```

Comment:
Any reason you're getting rid of this? Implementing in another patch?

Reply:
Nevermind, I see it moved up in the file

Reply:
Yes it needs to be handled before seccomp since seccomp will add/drop syscalls depending what is in the list for the container.